### DISCUSSION TITLE
[WiX4] ProgramFiles64Folder copies data to Program Files (x86)

### DISCUSSION THREAD

**Discussion by jan-tosovsky-cz** [wixtoolset/issues#6498] [Questions] [answered]
I've try to set target folder to these values:
- ProgramFiles64Folder
- ProgramFilesFolder
- ProgramFiles6432Folder (based on some mentions in GitHub issues)

Regardless this settings the package is installed always to Program Files (x86).

```xml
<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
    
    <Package
        Codepage="1250"
        Language="1029"
        Manufacturer="drifted.in"
        Name="JetBrowser"
        UpgradeCode="bd815300-19b5-4ce5-b8c6-1b592be2c574"
        Version="1.0.0"
        InstallerVersion="100"
        ProductCode="4700df0f-d076-4dd8-ae14-c4743e938e00">
        
        <SummaryInformation Description="JetBrowser 1.0 Installer" Manufacturer="drifted.in" Codepage="1250" />
        
        <Media Id="1" Cabinet="data.cab" EmbedCab="yes" />

        <StandardDirectory Id="ProgramFiles64Folder">
            <Directory Id="drifted.in" Name="drifted.in">
                <Directory Id="INSTALLDIR" Name="jetbrowser" />
            </Directory>
        </StandardDirectory>

        <DirectoryRef Id="INSTALLDIR">
            <Component Id="exe" Guid="5d56d614-add6-48d2-ae1f-3508a19d3626">
                <File Id="jetbrowser" Name="jetbrowser.exe" Source="jetbrowser.exe" KeyPath="yes" />
            </Component>
        </DirectoryRef>
        
        <Feature Id="MainApplication" Title="Main Application" Level="1">
            <ComponentRef Id="exe" />
        </Feature>

    </Package>
    
</Wix>
```
Should I specify it differently? Unfortunately, there is no documentation for StandardDirectory element, but there is [a request](https://github.com/wixtoolset/issues/issues/6496) already.

**Comment by BMurri** [comment, marked as answer]
Did you build a 64-bit package or a 32-bit package (what was your platform value on your build command-line)?

32-bit MSI packages, AFAICR, cannot place files in the 64-bit Program Files folder, but I may be remembering things incorrectly (it's been a few years that I've tried mucking with 64-bit platforms from 32-bit packages).

**Reply by jan-tosovsky-cz** [reply]
Can you provide an example of that cmdline switch? It is not listed when running `wix --help` command. Moreover suggested `wix build --help` (in help) returns `wix.exe : error WIX0118: Additional argument '--help' was unexpected.  Remove the argument and add the '-?' switch for more information.` When I finally run `wix build -?` it prints `TODO: Show build command help`.

**Reply by jcline-bonanza** [reply]
Following up on this answer as it helped me, the specific argument to use in the `wix build` command was `-arch x64`
